molecule/default/tests/test_default.py

```diff
@@ -12,7 +12,27 @@
 ).get_hosts("all")
 
 
-@pytest.mark.parametrize("x", [True])
-def test_packages(host, x):
-    """Run a dummy test, just to show what one would look like."""
-    assert x
+def test_tainum_installed(host):
```

Comment:
The `i` is in the wrong place in `tanium` in this def - I'd use suggest changes, but wanted to comment instead in case the typo is carried through elsewhere.

Reply:
Great catch!  See commit 61db149.

I also did a `find . -type f -exec grep -HniF tainum {} \;` and determined that this was the only place that suffered from this particular ailment.